custom_model_runner/datarobot_drum/drum/model_adapter.py

```diff
@@ -57,6 +58,20 @@ def __init__(self, model_dir, target_type=None):
         self._model_dir = model_dir
         self._target_type = target_type
 
+    @staticmethod
+    def _apply_sklearn_transformer(data, model):
+        try:
+            transformed = model.transform(data)
+            if type(transformed) == csr_matrix:
+                output_data = pd.DataFrame.sparse.from_spmatrix(transformed)
+            else:
```

Comment:
lets make sure its a dataframe or numpy array instead of a csc?

Reply:
since this is just applying a "raw" sklearn pipeline or transformer with no `transform` hook, I don't know that we want to constrain the output to dense formats? I guess we can explicitly tell users that they have to set sparse_threshold to 0 for column transformers, but that doesn't necessarily seem like a good idea if their transformation results in a massive sparse matrix....

Reply:
My comment refers to the fact that the transformed data could be a CSC or COO matrix instead of a CSR matrix, and the code will fail badly in those cases.